app/controllers/application_controller.rb

```diff
@@ -387,6 +388,16 @@ def report_data
     }
   end
 
+  def options_from_session(options)
```

Comment:
@himdel Please read the above comment https://github.com/ManageIQ/manageiq-ui-classic/pull/1553#discussion_r123006773 since it is applicable here.

> I suggest you use the same approach that already works with other nested lists, none of them need to load 7 things from session..

Those 7 things are what were originally passed to `nested_list` which `report_data` completely ignores. In fact, `options_from_session` makes sure that those options aren't forgotten. 
So once again, I think I have fixed a bug in `report_data` by providing it with data (from `session`) that was originally passed to `nested_list`/`get_view`

> (Except, where_clause would be problematic that way, but you don't really need to..)

I need the `where_clause` for this use case for the reasons I mentioned above.